include/server.hpp

```diff
@@ -8,39 +8,43 @@
 #include <string>
 #include <queue>
 #include <map>
+#include <QObject>
+
 #include "instrument.hpp"
 #include "note.hpp"
 #include "network_server.hpp"
 #include "synthesizer.hpp"
 
 
 
-class Server {
+class Server  : public QObject {
+
```

Comment:
if you emit signals from this class you will need the macro Q_OBJECT here

Reply:
They'll also need the `AUTOMOC` changes from `master`. I'll merge that tomorrow.